packages/core/src/lib/core/types-for-lists.ts

```diff
@@ -42,25 +44,37 @@ export type InitialisedField = Omit<NextFieldType, 'dbField' | 'access' | 'graph
   };
 };
 
-export type InitialisedList = {
+type CommonInitialisedList = {
   fields: Record<string, InitialisedField>;
   /** This will include the opposites to one-sided relationships */
   resolvedDbFields: Record<string, ResolvedDBField>;
-  pluralGraphQLName: string;
-  types: GraphQLTypesForList;
   access: ResolvedListAccessControl;
   hooks: ListHooks<BaseListTypeInfo>;
   adminUILabels: { label: string; singular: string; plural: string; path: string };
   cacheHint: ((args: CacheHintArgs) => CacheHint) | undefined;
-  maxResults: number;
   listKey: string;
-  lists: Record<string, InitialisedList>;
+  lists: Record<string, InitialisedSchema>;
   dbMap: string | undefined;
+  pluralGraphQLName: string;
+  types: GraphQLTypesForList;
+  maxResults: number;
   graphql: {
     isEnabled: IsEnabled;
   };
 };
```

Comment:
Many of the options for a 'common list' aren't actually common to singletons, we should review this, maybe separately to the first merge